django_graph_api/graphql/types.py

```diff
@@ -385,11 +385,26 @@ def __init__(self, object_type, **kwargs):
 
     @property
     def object_type(self):
+        def _str_to_class(s):
```

Comment:
Can this function be provided as a separate utility, instead of inside object_type?

Reply:
Per our discussion, leave it here to limit the use of this function as it should not be used in other places.